src/create-ad-slot.spec.ts

```diff
@@ -27,6 +28,7 @@ const DEFAULT_CONFIG = {
 		isSensitive: false,
 		section: 'uk-news',
 		videoDuration: 63,
+		edition: 'UK' as Edition,
```

Comment:
I think you could type these `as const` and TypeScript will figure out that this is a valid subtype of `Edition`. I'll leave that up to you though, I'm not sure there's very much in it.

```suggestion
		edition: 'UK' as const,
```

Reply:
You're right, thanks! I've made the change